spec.html

```diff
@@ -34274,7 +34274,7 @@ <h1>String.prototype [ @@iterator ] ( )</h1>
               1. Let _nextIndex_ be _position_ + _cp_.[[CodeUnitCount]].
               1. Let _resultString_ be the substring of _s_ from _position_ to _nextIndex_.
               1. Set _position_ to _nextIndex_.
-              1. Perform ? Yield(_resultString_).
+              1. Perform ? GeneratorYield(_resultString_).
```

Comment:
Don't these changes have to look like this?

```suggestion
              1. Perform ? GeneratorYield(! CreateIterResultObject(_resultString_, false)).
```

Reply:
Good catch, done.